source/tools/rapids-docker.md

```diff
@@ -1,6 +1,6 @@
-# Docker Images
+# Container Images
 
-RAPIDS docker images can be found on [Docker Hub](https://hub.docker.com/r/rapidsai/rapidsai).
+Installation instructions for Docker are hosted at the [RAPIDS Container Installation Docs Page](https://docs.rapids.ai/install#docker).
```

Comment:
The URL doesn't work ?

Reply:
Hrm good point. All the links on the `local.md` page seem broken too. @exactlyallan added this in #139, I wonder if that page has moved or something?

Reply:
Ok spoke to @exactlyallan we should be good to merge, that link will work later today.